### ISSUE TITLE
Multiple base colour choices

### ISSUE THREAD

**Issue by pb66** [emoncms/AndroidApp#58]
It would really make the app look better and easier to navigate if it was possible to choose a base colour for each page, perhaps a choice of a dozen or so basic colours would do rather than a complex palette or text colour code. Just to make it easily distinguishable when scrolling sideways between pages eg to stop at yellow for PV generation, for example.

**Comment by JumpMaster** [comment]
I would highly recommend using this ColorPicker.  Can be used as a dialog or within the preference fragment.

https://github.com/QuadFlask/colorpicker

**Comment by glynhudson** [comment]
Agree, this would be a really nice feature. Let's try and get the current V2 beta ready for production release (we're so close now), then we can turn our attention to new features.
